Uno.Gallery/Uno.Gallery.UWP/Views/Styles/TextBlock.xaml

```diff
@@ -0,0 +1,17 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+    <ResourceDictionary.MergedDictionaries>
+        <ResourceDictionary Source="../Fonts.xaml" />
+    </ResourceDictionary.MergedDictionaries>
+    
+	<Style x:Key="DefaultTexBlockStyle"
+           TargetType="TextBlock">
+
+        <Setter Property="FontFamily"
+				Value="{StaticResource RobotoFontFamily}" />
+    </Style>
+
+    <Style TargetType="TextBlock"
```

Comment:
Thats not going to be applied everywhere magically. Most of `TextBlock`s is using `Material\w+` style from material. We need to override the base style somehow, or make the modification in there.

Reply:
Maybe we could also copy everything the TextBlock.xaml into gallery. Add the font to the base default style, and then rename all the styles from `Material(\w+)` to `Gallery$1`, in every xaml except TextBlockSample.xaml.
This would get the take care of most of the TextBlocks we are using explicitly. Except:
- TextBlocks and Controls that uses TextBlock from material which we can keep as is; and
- a few that were implicitly based on some system fonts (text or glyph) which we need to review one by one.
  > eg: NavigationViewItem Content,

Reply:
Took the `TextBlock.xaml` from the `Uno.Material` repository and overrode the FontFamily on the Base Class used for all the Texblocks.